other/materialsproject/api_interoperability_showcase.py

```
@@ -46,25 +46,26 @@
         )
     )
 
-    get_ipython().system('GIT_BRANCH="dev"; export GIT_BRANCH; curl -s "https://raw.githubusercontent.com/Exabyte-io/api-examples/${GIT_BRANCH}/scripts/env.sh" | bash')
+    get_ipython().system(
```

Ruff (0.15.12)

[error] Undefined name `get_ipython`

(F821)